src/index.js

```diff
@@ -18,16 +18,28 @@ import {ApolloProvider} from "@apollo/react-hooks";
 import {setContext} from "apollo-link-context";
 
 import {setToken} from "actions/authActions";
+import * as actionCreators from "actions/actionTypes"
+import initialState from "./reducers/initialState";
+import invariant from 'redux-immutable-state-invariant';
 
+const composeEnhancers = composeWithDevTools({
+    actionCreators,
+    trace: true,
+    traceLimit: 25
+});
 
-const store = createStore(
-    rootReducer,
-    composeWithDevTools(applyMiddleware(thunk)),
-);
+// const store = createStore(
```

Comment:
Do we still need this store?

Reply:
this is a good point - I don't want to turn devtools on in production. so I'll add code here to switch between these 2 sections of code based on which env we're in.